cloud/customize-configuration.md

```diff
@@ -62,6 +62,17 @@ As with the basic database configuration parameters, any changes are highlighted
 and the `Apply changes` (or `Restart and apply changes`) button is available to
 click, prompting you to confirm any changes before the Service is modified.
 
+## Multiple databases
+Today, Timescale Cloud does not allow the creation of multiple databases within 
+a single service. If you require multiple databases, you will need to create a 
+service for each database. Another option would be to redesign your table 
+structure. Timescale Cloud does not support multiple databases within the same 
+service to reduce complexity. Each database would require its own installation 
+of TimescaleDB, managed independently, which breaks our ability to manage 
+maintenance upgrades for you. Supporting multiple databases also adds 
+significant complexity to the UI. To keep our services as easy to use as 
+possible, we have opted to limit services to one database per service.  
```

Comment:
```suggestion
To create more than one Timescale Cloud database, you need to create a new 
service for each database. Timescale Cloud does not support multiple 
databases within the same service. 
```

Reply:
I would really like to have some actual instructions for "redesign your table structure", because that's not really descriptive enough. I'm also not convinced we need all the internal reasons for designing it this way, it reads a little like "we found this difficult, so we didn't do it".

Reply:
Additionally, if the content is really just "use one service per db" then it should be an admonition, not under its own heading.

Reply:
Incorporated these changes into my next commit! Let me know if you feel this isn't sufficient. I agree that I am not sure that this warrants its own section.